cidr2ip/main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/fatih/color"
+	"github.com/ipinfo/cli/lib"
+	"github.com/ipinfo/cli/lib/complete/install"
+	"github.com/spf13/pflag"
+)
+
+var progBase = filepath.Base(os.Args[0])
+var version = "1.0.0"
+
+func printHelp() {
+	fmt.Printf(
+		`Usage: %s [<opts>] <cidrs | filepath>
+
+Description:
+	Accepts CIDRs and file paths to files containing CIDRs, converting
+	them all to individual IPs within those CIDRs.
```

Comment:
Please replace all tab characters being used in help message strings with raw space characters only, otherwise it prints in a messed up way for users.